app/client/src/modules/app/organization/organization.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+module.exports =
+    angular.module('permissive.organization', [])
+    .config(function ($stateProvider) {
+        $stateProvider
+            .state('organization', {
+                url: '/organization',
+                templateUrl: 'app/organization/organization.html',
+                controller: 'organizationController as ctrl'
+            });
+    })
+    .controller('organizationController', function ($http, organizationService) {
+        console.log('getting organization');
+        organizationService.query().$promise.then(org => {
+            this.users = org.users;
+            delete org.users;
+            let organization = [];
+            angular.forEach(org, (org, key) => {
```

Comment:
This comment is just expressing a preference for native methods wherever possible, but if you think the angular wrapper is cleaner, that's fine.